Appy.GitDb.Tests/Merging.cs

```diff
@@ -65,8 +77,14 @@ public class MergingWhenThereAreNoConflicts : WithRepo
                 _addedFiles.Remove(i);
                 _removedFiles.Add(i);
             }
-        }
+        }       
 
+        [Fact]
+        public void MergesShouldSuccedWithValidInfo()
+        {
+            _firstMergeResult.ShouldBeEquivalentTo(_firstMergeExpected);
+            _secondMergeResult.ShouldBeEquivalentTo(_secondMergeExpected);
```

Comment:
- Inline the expectations here, so it's more visible what the result should be
- Split the test into 2 so the failure is clearer